packages/next/src/client/components/react-dev-overlay/_experimental/internal/components/dialog/styles.ts

```diff
@@ -5,9 +5,10 @@ const styles = css`
     --next-dialog-radius: var(--rounded-xl);
     --next-dialog-footer-height: 48px;
     --next-dialog-max-width: 960px;
+    --next-dialog-row-padding: 16px;
 
     display: flex;
-    flex-direction: column;
+    flex-direction: column-reverse;
```

Comment:
Reversed so we can leverage natural DOM order to keep the notch navigation a layer above the dialog contents on the Z-axis without explicitly declaring `z-index`